doc/ReleaseNotes.md

```diff
@@ -2,49 +2,148 @@
+  for applications, it is recommended that the Python venv module is used to
+  manage dependencies rather than relying on modules provided with the
+  operating system.
+
+* A patch that was previously applied to OpenSSH to resolve problems for
+  VMWare guests using NAT mode has been removed. OpenSSH will now use
+  standard DSCP options for QoS.
+
+* The `ec2-credential` and `azure-agent` packages have been removed in favour
+  of `cloud-init`. If you upgrade from a previous release which has these
+  packages installed, they will be retained.
+
+* A new tuneable kernel parameter has been added for `ipf` that allows response
+  to outbound packets.
+  ```
+  echo 'set ipf:outboundrst = 1' > /etc/system.d/ipf
+  ```
+  This then allows the use of outbound rules using actions such as `return-rst`.
+
+* The console driver now supports hiding the cursor and has received some
+  improvements around colour and emboldening support.
+
 ### Commands and Command Options
 
+* The `pwait` utility will now never wait on its own process ID.
+
+* `format` will now recognise a new (unformatted) NVMe disk.
+
+* The `strip` command could previously become confused when processing
+  binaries containing a lot of debug sections.
+
+* `psrinfo` would sometimes fail to emit the processor socket type, even if
+  it knows it.
+
+* `pcieadm` decoded the address of an expansion ROM incorrectly.
+
+* The `pcieadm` utility has been enhanced with support for decoding 32.0GT/s
+  speeds and NPEM capabilities.
+
 ### Libraries and Library Functions
 
-### Zones
+* The [getifaddrs(3SOCKET)](https://man.omnios.org/getifaddrs) library
+  function has been extended to enumerate more address families.
 
 ### LX zones
 
+* It is now possible to specify multiple IP addresses per VNIC in an lx
+  zone configuration.
+
 ### Bhyve
 
-### ZFS
+* Many updates and improvements to bhyve from upstream illumos.
+
+* The `viona` driver now supports a control queue and promiscuous mode.
+
+* The `::sysregs` command under `mdb -b` did not work on AMD, this has been
+  resolved.
 
 ### Package Management
 
+* The format of the OmniOS IPS repositories for this release has changed so
+  that catalogues are stored in native UTF-8 format (see
+  [pkgrepo(1)](https://man.omnios.org/pkgrepo#repository/format)). This allows
+  clients such as pkg(1) to parse the catalogues faster and with less memory
+  overhead but means that these repositories are not compatible with pkg
+  clients older than that found in the r151038 release.
+
 ### Hardware Support
 
+* PCIe hotplug is now supported for slots without a power controller.
+
+* A new AMD Unified Memory Controller driver is available.
+
+* Additional AVX512 capabilities are now supported.
+
+* SMBIOS 3.6 support has been added.
+
+* Block devices now support GUIDs for device naming. This may cause some
+  system devices to change their device path on first boot with this release.
```

Comment:
what is the effect of this ? how will zfs cope with this ?

Reply:
ZFS should be fine with this - and has been on all of the systems we've tested. Just a few extra messages on the console on that first boot. Rolling back to older BEs works fine too.
This is different to the other change which we've held back until r151045, which adds new device ID types and makes rollback harder.

Reply:
aah, ok, thought this was the full monty!